iphone/Maps/Classes/MapViewController.mm

```diff
@@ -709,14 +709,22 @@ - (BOOL)canBecomeFirstResponder {
 }
 
 - (NSArray *)keyCommands {
-   return @[[UIKeyCommand keyCommandWithInput:UIKeyInputDownArrow modifierFlags:0 action:@selector(zoomOut)], // Alternative, not shown when holding CMD
```

Comment:
You can create an array in the old way, in was more succinct.

Reply:
Thanks for the feedback, I have made the adjustment.